### ISSUE TITLE
Windows SmartScreen blocks batch files

### ISSUE THREAD

**Issue by Kenny2github** [smelt-modding/smelt-modding.github.io#1]
#### Expected behavior
One should be able to use this system anytime
#### Actual behavior
Windows SmartScreen blocks setup.bat from running
#### Steps to reproduce
Try to run setup.bat after downloading
#### Windows version
Microsoft Windows [Version 10.0.10586]
#### Workaround
1. Right click setup.bat and choose "Properties".
2. Check the box that says "Unblock" and click OK.

**Comment by Kenny2github** [comment]
Marking as `help wanted` since the general public might be able to help

**Comment by VutonDesign** [comment]
@Kenny2github I don't know if this could be fixed. 😕

**Comment by radioblahaj** [comment]
Is this promblem in permsiions?

**Comment by Kenny2github** [comment]
@redmangospros no, the problem is Windows blocks batch files from the Internet - but there's no way to make it stop doing that.

**Comment by radioblahaj** [comment]
Well what do you.mean. you can set this ip in settings to allow donloads from tbe ontwrnet.
I did that.

**Comment by Kenny2github** [comment]
@redmangospros that's true, but for others it doesn't work that way, so this is still a problem.

**Comment by radioblahaj** [comment]
Oh okay

**Comment by ghost** [comment]
This is intentional because random batch files can do bad things to your computer.
